environment.yml

```diff
@@ -0,0 +1,66 @@
+name: cognoma-cancer-data
+dependencies:
```

Comment:
@clairemcleod interesting error. It looks like `zeromq 4.1.4` is available for Linux, which I used to make the PR, but not for OS X in the [anaconda channel](https://anaconda.org/anaconda/zeromq/files?page=1). 4.1.3 is available for OS X.

It looks like others are having [similar issues](http://stackoverflow.com/q/38400676/4651668) recently. Options are to downgrade the versions for OS X compatibility, or to notify anaconda and see if they can fix this.